service.watchedlist/resources/language/English/strings.po

```diff
@@ -0,0 +1,416 @@
+# XBMC Media Center language file
```

Comment:
please adjust to the new language folder layout `/resources/language/resource.language.en_gb/strings.po`